STACK-UP-3/warriorz-backend
# #171784073 Setup Sequelize

#### What does this PR do?
- It adds  configuration for setting up sequelize
#### Description of Task to be completed?
- We are going to use sequlize as our ORM on the API. Set up the migrations and models structure, have a dummy user table and model setup to test the whole configuration. Add the various required setup in the README if any is required.
#### How should this be manually tested?
- Clone the repo and checkout to ch-sequelize-setup-171784073
- Check the config file in the server root folder.
#### What are the relevant pivotal tracker stories?
- [171784073](https://www.pivotaltracker.com/story/show/171784073)
